tox.ini

```diff
@@ -1,19 +1,60 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
 envlist =
-    py27,pypy,py34,py35,py36,py37,pypy3,coverage
+    lint,
+    py27,
+    pypy,
+    py35,
+    py36,
+    py37,
+    py38,
+    pypy3,
+    coverage
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
     zope-testrunner --test-path=src []
-deps =
-    .[test]
+extras = test
 
+[testenv:lint]
+basepython = python3
+skip_install = true
+deps =
+    flake8
+    check-manifest
+    check-python-versions
+commands =
+    flake8 src setup.py
+    check-manifest
+    check-python-versions .
```

Comment:
FWIW `.` is the default location for `check-python-versions` and doesn't need to be explicitly specified.

Reply:
Thank you for the info I am going to remove it.